.lintrunner.toml

```diff
@@ -0,0 +1,285 @@
+# Configuration for lintrunner https://github.com/suo/lintrunner
+
+[[linter]]
+code = 'FLAKE8'
+include_patterns = [
+    '**/*.py',
+]
+exclude_patterns = [
+    '*_pb2*',
+    '.setuptools-cmake-build/*',
+    'docs/**',
+]
+command = [
+    'python3',
+    '-m',
+    'lintrunner_adapters',
+    'run',
+    'flake8_linter',
+    '--',
+    '@{{PATHSFILE}}'
+]
+init_command = [
+    'python3',
+    '-m',
+    'lintrunner_adapters',
+    'run',
+    'pip_init',
+    '--dry-run={{DRYRUN}}',
+    'flake8==5.0.4',
+    'flake8-bugbear==22.10.27',
+    'flake8-pyi==22.10.0',
+    'dlint==0.13.0',
+]
+
+[[linter]]
+code = 'MYPY'
+include_patterns = [
+    'onnx/**/*.py',
+]
+exclude_patterns = [
+    '**/*.pyi',
+    'onnx/backend/test/**',
+    'onnx/reference/ops/**',  # FIXME: Enable this once typing is fixed
+    'onnx/test/parser_test.py',
+    'onnx/test/numpy_helper_test.py',
+    'onnx/reference/reference_evaluator.py',
+]
+command = [
+    'python3',
+    '-m',
+    'lintrunner_adapters',
+    'run',
+    'mypy_linter',
+    '--config=pyproject.toml',
+    '--',
+    '@{{PATHSFILE}}'
+]
+init_command = [
+    'python3',
+    '-m',
+    'lintrunner_adapters',
+    'run',
+    'pip_init',
+    '--dry-run={{DRYRUN}}',
+    'mypy==0.971',
+    'types-protobuf==3.18.4',
+]
+
+# TODO: Enable NOQA
```

Comment:
Could you please provide more context regarding this TODO (for instance, what is the blocker that we cannot apply it now)? Shall we create an issue to track it?

Reply:
Done. Added inline and https://github.com/onnx/onnx/issues/4664